app/forms/onboarding/request_form.rb

```diff
@@ -49,8 +49,6 @@ def proxy_auth_params
   end
 
   def test_api!
-    return unless ProxyDeploymentService.call(model, v1_compatible: true)
```

Comment:
I don't think you should be removing this line, just the argument.

Reply:
You are right see https://github.com/3scale/porta/pull/2495/files#r660448399